Resources/ServerInfo/_Moffstation/Guidebook/ServerRules/MoffR3Powergaming.xml

```diff
@@ -0,0 +1,32 @@
+<Document>
+  # Section 3: Powergaming
+
+  3.1 Do not powergame. That is, minmaxing to try and achieve the best possible results to the point of being detrimental to roleplay, rushing to acquire strong or useful equipment you do not need, performing meta tricks and exploits that are unfair or unfit for a roleplay environment, and so forth.
```

Comment:
Could do with some clarification as to what is fair and not fair for power-gaming. Something along the lines of "is what my character doing something they would have a reason to do?" The wording is a bit vague as-is.